### PR TITLE
auth-add-playList

### PR DESCRIPTION
**Pull request by theVivek-kumar**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
         <Route path='/' element={<LandingPage/>}/>
-        <Route path='/videoListing' element={<VideoListing/>}/>
+        <Route path='/videoListing' element={<VideoListing />} />
         <Route path='/singleVideoPage/:videoId' element={<SingleVideoPage />} />
-        <Route path='/watchLater' element={<WatchLaterpage />} />
-        <Route path='/historyPage' element={<HistoryPage />} />
-        <Route path='/likeVideo' element={<LikeVideoPage />} />
+        {/* private-route */}
+
+        <Route path='/watchLater' element={<AuthComp><WatchLaterpage /></AuthComp>} />
```
Comment (Adarsh-Patel-Dev) [inline src/App.js:26]: Instead of wrapping each private route with `<AuthComp>` You can use Outlet

File: src/components/login/Login.js [hunk 2]
```diff
@@ -16,12 +16,12 @@ function Login() {
               <input className="input-lg" type="password" placeholder="************"/>
               <div class="input-check-btn">
                 <input className="input-checkbox" type="checkbox"/>
-                <h3>Remember me</h3>
-                <span className="frg-paswrd"> Forgot your Password?</span>
+                <h4>Remember me</h4>
+                <span  className="frg-paswrd"> Forgot your Password?</span>
```
Comment (Adarsh-Patel-Dev) [inline src/components/login/Login.js:20]: ClassName can be better

File: src/components/login/login.css [hunk 3]
```diff
@@ -35,7 +35,7 @@
     width: 30rem;
     height: 4rem;
     border-radius: 5px;
-    text-decoration: none;
+    text-decoration: none!important;
```
Comment (Adarsh-Patel-Dev) [inline src/components/login/login.css:38]: try to avoid `!important` in css

**Review by Adarsh-Patel-Dev — APPROVED** [review]
Suggested some minor changes
Also you should add some toast/alerts to show respective action

**Review by MaruthiPothuganti — COMMENTED** [review]
UI and Code looks good. Can go ahead and merge.

**Pull request merged**